packages/gamut-labs/src/LandingPage/PageHero.tsx

```diff
@@ -91,13 +93,18 @@ export const PageHero: React.FC<PageHeroProps> = ({
         )}
       </Column>
       {media && (
-        <Column size={{ sm: right }}>
+        <Column
+          size={{ sm: right }}
+          gridRowStart={showImageOnMobile ? { _: 1, sm: 'initial' } : undefined}
+        >
           {media.type === 'image' ? (
             <Image
               src={media.src}
               alt={media.alt}
               width={1}
-              display={{ _: 'none', sm: 'initial' }}
+              display={
+                showImageOnMobile ? undefined : { _: 'none', sm: 'initial' }
+              }
```

Comment:
NITTIEST NIT OF ALL TIME

```suggestion
              display={{
                 _: showImageOnMobile ? 'initial' : 'none' , 
                 sm: 'initial' 
               }}
```

Reply:
Hi Aaron, I ported this suggestion into a new branch https://github.com/Codecademy/client-modules/pull/2085/files